Source/EventFlow.Sql/Integrations/TableParameter.cs

```diff
@@ -31,7 +32,7 @@
 using Dapper;
 using Microsoft.SqlServer.Server;
```

Comment:
@JC008 Very, very good point... Then I'll have to reintroduce the EventFlow.MsSql package once again and move the table parameter to that.